gcd/src/main/java/org/spine3/server/storage/datastore/DatastoreStorageFactory.java

```diff
@@ -142,11 +143,13 @@ public StandStorage createStandStorage() {
     public <I> RecordStorage<I> createRecordStorage(Class<? extends Entity<I, ?>> entityClass) {
         final Class<Message> messageClass = getGenericParameterType(entityClass, ENTITY_MESSAGE_TYPE_PARAMETER_INDEX);
         final TypeUrl typeUrl = TypeUrl.of(messageClass);
+        final Class<I> idClass = Classes.getGenericParameterType(entityClass, ID.getIndex());
         final Descriptor descriptor = (Descriptor) typeUrl.getDescriptor();
         final DsRecordStorage<I> result = new DsRecordStorage<>(descriptor,
```

Comment:
Let's introduce a builder for the `DsRecordStorage` in a separate PR. There are too many arguments.